docs/UserGuide.md

```diff
@@ -72,6 +72,8 @@ List of `<optional arguments>`:
 🚩: Duplicate entry of tasks are **allowed**. 
 
 
+🚩: Duplicate task descriptions are allowed. 
+
```

Comment:
This line not required anymore

![image](https://user-images.githubusercontent.com/42372471/98431451-70b3bf00-20f0-11eb-9f3d-db6cb0844074.png)

Reply:
Fixed